lib/spack/spack/test/python_version.py

```diff
@@ -89,14 +89,15 @@ def pyfiles(search_paths, exclude=()):
                     yield os.path.join(root, filename)
 
 
-def check_python_versions(files):
+def check_python_versions(files, warn_only_py26=False):
```

Comment:
Mucking around in core will make future merges harder. I'm actually trying to move away from all the workarounds we have in there.

Reply:
In the next change they do to this file I do hope they get rid of python 2.6 compatibility enforcement. Do you suggest some lighter changes?